src/olympia/devhub/forms.py

```diff
@@ -342,11 +342,21 @@ def __init__(self, *args, **kw):
 
     def clean_contributions(self):
         if self.cleaned_data['contributions']:
-            hostname = urlsplit(self.cleaned_data['contributions']).hostname
+            parsed_url = urlsplit(self.cleaned_data['contributions'])
+            hostname = parsed_url.hostname
+            path = parsed_url.path
+
             if not hostname.endswith(amo.VALID_CONTRIBUTION_DOMAINS):
                 raise forms.ValidationError(ugettext(
                     'URL domain must be one of [%s], or a subdomain.'
                 ) % ', '.join(amo.VALID_CONTRIBUTION_DOMAINS))
+            elif (hostname == 'github.com' and
```

Comment:
I wonder if it's worth making `github.com` into a constant, so that it can be used here, in `src/olympia/constants/base.py`, and also in the tests, without having to hardcode it multiple times? It would also allow for it to be changed easily if for some reason the GitHub hostname changed.

@eviljeff | @diox WDYT?

Reply:
I don't think it's really worth the effort.

Reply:
Fair enough. This is good to land, then. Thanks for the patch @conema!